theano/sandbox/cuda/tests/test_cusolver.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import absolute_import, division, print_function
+
+import unittest
+import numpy
+import theano
+
+from theano.tests import unittest_tools as utt
+
+# Skip tests if cuda_ndarray is not available.
+from nose.plugins.skip import SkipTest
+import theano.sandbox.cuda as cuda_ndarray
+from theano.misc.pycuda_init import pycuda_available
+from theano.sandbox.cuda.cusolver import cusolver_available
+
+from theano.sandbox.cuda import cusolver
+
+if not cuda_ndarray.cuda_available:
```

Comment:
travis is failing as the tests aren't skipped. I don't understand why. I'll close/reopen travis again to make travis run with the current master of Theano.